docs/en_US/database_dialog.rst

```diff
@@ -44,8 +44,8 @@ Use the *Definition* tab to set properties for the database:
 * Select the strategy from the drop-down listbox in the *Strategy* field while
   creating a new database. This option is available from v15 and above.
 * Select the locale provider from the drop-down listbox in the *Locale Provider*
-  field to set the default collation for this database. Possible values are: icu, libc.
-  This option is available from v15 and above.
+  field to set the default collation for this database. Possible values are: icu, libc and builtin.
+  This option is available from v15 and above. **Note:** builtin option is available from v17 onwards.
```

Comment:
This could be changed to: 
The icu and libc options are available from v15 and builtin option is available from v17 onwards.